.bazelignore

```diff
@@ -1 +1,2 @@
 # NB: don't ignore the modules/ folder as it contains inputs to bazel test targets.
+tools/bzlmod_migration_test_examples
```

Comment:
can you fix all the line ending issues by adding a new line in such files?

Reply:
Done